### DISCUSSION TITLE
Is there a way to create split slides?

### DISCUSSION THREAD

**Discussion by fuine** [touying-typ/touying#97] [Q&A] [answered]
This is an idea taken from [marp](https://marpit.marp.app/image-syntax?id=split-backgrounds) and looks like this:
![split-background](https://github.com/user-attachments/assets/11d0ab3d-bd4b-40ae-aa3e-2d66f1c365df)

I can replicate it in `polylux` but I'm not sure if it's attainable in `touying`. As far as I understand it, it would require offsetting header, content and footer and I'm not sure if it's doable in `typst`

**Comment by OrangeX4** [comment]
Use something like:

```typst
#empty-slide(
  config: config-page(
    background: align(left, rect(height: 100%, width: 50%, fill: red)),
    margin: (left: 50% + 1em),
  ),
)[
  Content.
]
```

![image](https://github.com/user-attachments/assets/2d090ed3-7629-4377-87d6-da9d4d759cfa)

**Reply by fuine** [reply]
Ok, and then I guess I would need to render the header/footer in the content?

**Reply by OrangeX4** [reply, marked as answer]
```typst
#slide(config: config-page(
  background: align(left, rect(height: 100%, width: 50%, fill: red)),
  margin: (left: 50%, right: 0em),
) + config-common(zero-margin-header: false, zero-margin-footer: false))[
  #pad(1em)[
    Content.
  ]
]
```

![image](https://github.com/user-attachments/assets/66e69233-f1a9-4ea4-84c2-ec784d4f0c26)

**Reply by fuine** [reply]
That's great, thank you very much!

**Comment by OrangeX4** [comment]
or

```typst
#empty-slide(config: config-page(margin: 0em))[
  #rect(height: 100%, width: 100%, fill: red)
][
  #pad(2em)[
    Content.
  ]
]
```
